unicode-transforms.cabal

```diff
@@ -75,6 +74,9 @@ library
       Data.Unicode.Types
       Data.Text.Normalize
 
+      -- Internal
+      Data.Unicode.Internal.Char.DerivedNormalizationProperties
```

Comment:
This can go in the other modules section below.

Reply:
Not possible, because it is used in tests.